### PR TITLE
Display apache license notice on startup

### LINKED ISSUE #3037: Print info about Apache 2 licence at startup
Following discussion with legal, it would be prudent for Teku to print a notice to the console and the logs at startup to the effect that its use is governed by the Apache 2 licence. This covers the case of someone directly downloading a binary or docker image without going via our docs or GitHub.

Suggested text:
```
This software is licensed under the Apache License, Version 2.0 (the "License");
you may not use this software except in compliance with the License.
You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
```

### PR DESCRIPTION
**Pull request by ajsutton**
## PR Description
Display an license notice on startup.

Note: Deliberately logs the notice on a single line because multiline log messages are generally frowned upon.

## Fixed Issue(s)
fixes #3037 

## Documentation

- [x] I thought about documentation and added the `documentation` label to this PR if updates are required.

### CODE REVIEW

File: infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java [hunk 1]
```diff
@@ -38,6 +38,10 @@ private StatusLogger(final String name) {
 
   public void onStartup(final String version) {
     log.info("Teku version: {}", version);
+    log.info(
```
Comment (rolfyone) [inline infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java:41]: just wonder if we should bump this to a warning...

**Review by rolfyone — APPROVED** [review]
LGTM

File: infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java (hunk 1 above)
Comment (ajsutton) [reply infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java:41]: I don't think there's a better example of an informational message. :) Warn implies something the user should be concerned about but this is just routine information they may need to know.

Comment (rolfyone) [reply infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java:41]: who pays attention to info messages.... :)

**Pull request merged**